### PR TITLE
added parameter passing to application via extras. Added copying of s…

### PR DESCRIPTION
**Pull request by amerhifi**
https://highfidelity.manuscript.com/f/cases/19113/Debug-mode-Jump-to-a-domain-on-start
#9113 …erverless files in the resources cmake. Originally from gcaleros pr

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16680/HighFidelity-Beta-PR14761-943253b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16680/HighFidelity-Beta-Interface-PR14761-943253b.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16683/HighFidelity-Beta-PR14761-1c6c8e1.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16680/HighFidelity-Beta-PR14761-943253b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16680/HighFidelity-Beta-Interface-PR14761-943253b.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16685/HighFidelity-Beta-PR14761-b5d6cf9.apk

**Comment by amerhifi** [comment]
**Testing**
 
Ability to load and execute scripts from command line on Android.

**Process**

Run the app using adb with the following command:

adb shell am start -n io.highfidelity.hifiinterface/.PermissionChecker --es args \"--url file:///~/serverless/tutorial.json --no-updater --no-login-suggestion --testScript <ENTER SCRIPT URL.js> quitWhenFinished --testResultsLocation /sdcard/snapshots\"

watch logcat for any scripting errors.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16685/HighFidelity-Beta-PR14761-b5d6cf9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16685/HighFidelity-Beta-Interface-PR14761-b5d6cf9.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16685/HighFidelity-Beta-PR14761-b5d6cf9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16685/HighFidelity-Beta-Interface-PR14761-b5d6cf9.exe

File: interface/CMakeLists.txt [hunk 1]
```diff
@@ -330,6 +330,10 @@ if (APPLE)
     COMMAND "${CMAKE_COMMAND}" -E copy_directory
       "${PROJECT_SOURCE_DIR}/resources/fonts"
       "${RESOURCES_DEV_DIR}/fonts"
+      #copy serverless for
```
Comment (NissimHadar) [inline interface/CMakeLists.txt]: Typo?

**Comment by amerhifi** [comment]
https://highfidelity.manuscript.com/f/cases/19113/Debug-mode-Jump-to-a-domain-on-start

**Comment by qligmaloney** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16855/HighFidelity-Beta-PR14761-b5d6cf9.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16855/HighFidelity-Beta-PR14761-b5d6cf9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16855/HighFidelity-Beta-Interface-PR14761-b5d6cf9.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16855/HighFidelity-Beta-PR14761-b5d6cf9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16855/HighFidelity-Beta-Interface-PR14761-b5d6cf9.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16869/HighFidelity-Beta-PR14761-7a254d8.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16869/HighFidelity-Beta-PR14761-7a254d8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16869/HighFidelity-Beta-Interface-PR14761-7a254d8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16869/HighFidelity-Beta-PR14761-7a254d8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16869/HighFidelity-Beta-Interface-PR14761-7a254d8.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16886/HighFidelity-Beta-PR14761-f4eeff5.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16886/HighFidelity-Beta-PR14761-f4eeff5.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16886/HighFidelity-Beta-Interface-PR14761-f4eeff5.dmg

**Pull request closed without merging**